pkg/controller/performanceprofile/components/utils.go

```diff
@@ -27,15 +29,64 @@ func SplitLabelKey(s string) (domain, role string, err error) {
 	return parts[0], parts[1], nil
 }
 
+func cpusListToArray(cpusListStr string) ([]int, error) {
```

Comment:
if we can vendor kubernetes modules, you may find this useful:
`k8s.io/kubernetes/pkg/kubelet/cm/cpuset`
sorry for late noticing

Reply:
nice, it also does not have other k8s dependencies except the klog

Reply:
we still can use the package mentioned by @fromanirh, but we can do it in the following PR